wrap_js/__test__/AddSignaturesToCet.spec.ts

```diff
@@ -6,16 +6,18 @@ const testCase = [
   TestHelper.createTestCase(
     "AddSignaturesToCet",
     cfddlcjs.AddSignaturesToCet,
-    {
-      cetHex: TestData.CetHexUnsigned,
-      localFundPubkey: TestData.LocalFundPubkey,
-      remoteFundPubkey: TestData.RemoteFundPubkey,
-      fundTxId: TestData.FundTxId,
-      signatures: [
-        TestData.CetLocalRawSignature,
-        TestData.CetRemoteRawSignature,
-      ],
-    },
+    TestData.Initialize().then(() => {
```

Comment:
Instead of calling `Initialize` everywhere, I think it would be better to just return a (non async) function like:
```typescript
    () => {
      return {
        cetHex: TestData.CetHexUnsigned,
        localFundPubkey: TestData.LocalFundPubkey,
        remoteFundPubkey: TestData.RemoteFundPubkey,
        fundTxId: TestData.FundTxId,
        signatures: [
          TestData.CetLocalRawSignature,
          TestData.CetRemoteRawSignature,
        ],
      };
    },
```

Then in TestHelper:
```typescript
testCases.forEach((testCase) => {
      describe(testDescribe, () => {
        beforeAll(async () => await Initialize());
        beforeEach(() => testCase.setupFunction());
        it(testCase.testCaseName, async () => {
          const expected = testCase.expected;
          let received: TResponse;
          try {
            const req: TRequest = testCase.testRequest();
            received = testCase.testFunction(req);
          } catch (error) {
            received = JSON.parse(error.message);
          }
          if (testCase.convertFunction) {
            received = testCase.convertFunction(received);
          }
          if (Array.isArray(expected)) {
            expect(expected).toContain(received);
          } else {
            expect(received).toEqual(expected);
          }
        });
        afterEach(() => testCase.tearDownFunction());
      });
    });
```

What do you think? (I feel it makes the code a bit lighter)

Reply:
This seems better! 👍  I'll try!